htu21d.py

```diff
@@ -43,30 +44,30 @@ def _crc_check(self, value):
 
     def _issue_measurement(self, write_address):
         """Issue a measurement.
-
         Args:
             write_address (int): address to write to
         :return:
         """
+        import time
         self.i2c.start()
         self.i2c.writeto_mem(int(self.ADDRESS), int(write_address), '')
         self.i2c.stop()
         data = bytearray(3)
+        time.sleep_ms(50)
         self.i2c.readfrom_into(self.ADDRESS, data)
         if not self._crc_check(data):
             raise ValueError()
         raw = (data[0] << 8) + data[1]
         raw &= 0xFFFC
-        return raw
-
-    @property
```

Comment:
why do you issue an api change here and remove the property decorator?

Reply:
This is the first library for a temperature sensor that I have seen using a property for temperature and humidity. that alone was a reason for me to change the api, so it is similar to the other libraries. 
Also working towards an async api meant dropping the property as this is not supported with asyncio.